### PR TITLE
Adding celery for multiple versions in tox.

### PR DESCRIPTION
**Pull request by awais786**
- Adding celery 5.0 testing using tox.
- Drop py3.5 testing
- execute make upgrade using py38 with celery<5 constraint.
- replaced task decorator with shared decorator.

In next PR I will execute the make upgrade with celery==5.0.2

I am making a sandbox to do some testing using this branch https://github.com/edx/edx-platform/pull/25835

Steps to enable event on sandbox.

1. Copy this config in lms/env/common.py 
https://gist.github.com/awais786/56b078e81b3879b63b419228f8620971

2.  https://webhook.site/  go here and make new URL.

3. https://awais786.sandbox.edx.org/admin/event_routing_backends/routerconfiguration/add/

backend name = `caliper`
values = `[{"match_params":{"context.host":"awais786.sandbox.edx.org"},"router_type":"AUTH_HEADERS","host_configurations":{"url":"https://webhook.site/0b645ad4-82f2-455c-8cf3-42c07b14c507"}}]`

4. sudo /edx/bin/supervisorctl restart all
5. Now register new user and enroll celery flower will show the tasks and these tasks will appear on webhooksite as well.

### CODE REVIEW

File: eventtracking/tasks.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import json
 
 from celery.utils.log import get_task_logger
-from celery import task
```
Comment (awais786) [inline eventtracking/tasks.py:7]: Its removed in celery5.0 so updating it.  Its compatible with 4.4.7 and 5.0 also.

File: setup.py [hunk 2]
```diff
@@ -50,7 +50,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='event-tracking',
-    version='1.0.0',
+    version='2.0.0',
```
Comment (jmbowman) [inline setup.py:53]: The changes don't seem to justify a major version bump; maybe 1.0.1 instead?

**Comment by awais786** [comment]
Screen Shot 2020-12-14 at 12 48 54 PM

**Comment by awais786** [comment]
Screen Shot 2020-12-14 at 12 50 21 PM

**Review by mzulqarnain1 — APPROVED** [review]

**Pull request merged**